OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -625,21 +626,21 @@ hasStateStorage() const
 //-----------------------------------------------------------------------------
 
 bool Manager::
-integrate(SimTK::State& s, double finalTime)
+integrate(double finalTime)
 {
     int step = 0; // for AnalysisSet::step()
 
-    if (!_integ) {
-        throw Exception("Manager::integrate(): "
-            "Integrator has not been set. Construct the Manager "
-            "with an integrator, or call Manager::setIntegrator().");
+    if (_timeStepper == NULL) {
```

Comment:
Nitpicky: it would be good to transition to using `nullptr` instead of `NULL`.

Reply:
Nitpicky again: I just noticed this is a a smart pointer; I think you could just do `if (_timeStepper)`. Feel free to ignore these comments though.